pkgs/firecracker-microvm/firectl/registry.yaml

```diff
@@ -0,0 +1,13 @@
+packages:
+  - type: github_release
+    repo_owner: firecracker-microvm
+    repo_name: firectl
+    description: firectl is a command-line tool to run Firecracker microVMs
+    asset: firectl-{{ .Version }}
+    format: raw
+    supported_envs:
+      - linux
+    checksum:
+      type: github_release
+      asset: "{{.Asset}}.sha256"
+      algorithm: sha256
```

Comment:
Please add a newline at the end of file.

<img width="299" alt="image" src="https://github.com/user-attachments/assets/bf88a26c-7a67-416f-b5e1-43f76a759611">

If you use VSCode, I recommend setting `"files.insertFinalNewline": true`.

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Fixed. 7d279858d14102c3d4660b64d972f9e8aab05c7e